src/ukify/ukify.py

```diff
@@ -406,17 +452,20 @@ def add_section(self, section: Section) -> None:
 
 class SignTool:
     @staticmethod
-    def sign(input_f: str, output_f: str, opts: argparse.Namespace) -> None:
+    def sign(input_f: str, output_f: str, opts: UkifyConfig) -> None:
         raise NotImplementedError()
 
     @staticmethod
-    def verify(opts: argparse.Namespace) -> bool:
+    def verify(opts: UkifyConfig) -> bool:
         raise NotImplementedError()
 
 
 class PeSign(SignTool):
     @staticmethod
-    def sign(input_f: str, output_f: str, opts: argparse.Namespace) -> None:
+    def sign(input_f: str, output_f: str, opts: UkifyConfig) -> None:
+        assert opts.sb_certdir is not None
+        assert opts.sb_cert_name is not None
```

Comment:
Ah, you added them in the last commit. Maybe, better to move to the earlier commit.

Reply:
I moved the asserts into the commit changing the interface, since that's also where I wanted them, but lost it in a rebase.